src/Scripting/CSharp/CSharpScripting.csproj

```diff
@@ -51,6 +51,5 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.CSharp" Version="$(MicrosoftCSharpVersion)" />
-    <PackageReference Include="System.Dynamic.Runtime" Version="$(SystemDynamicRuntimeVersion)" />
```

Comment:
❓ Can you remove any of these from packages.props?

Reply:
Eventually yes. I'm trying to get BuildBoss back to the state where it will gripe if you have Packages.props entries that aren't valid. I'm still unwinding the string here though on our core package references though. Have to track @terrajobst down to get some more detail before I can squash the rest of this.

Reply:
There are also other implications like: what if you use the wrong version of Sysetm.Dynamic.Runtime? We are kind of taking it on faith that we're doing it correctly today vs. using the correct versions.

Reply:
> Have to track @terrajobst down to get some more detail before I can squash the rest of this. 

What concerns do you have around this?